roles/installer/tasks/database_configuration.yml

```diff
@@ -85,21 +85,21 @@
 
 - name: Set PostgreSQL Configuration
   set_fact:
+    __postgres_configuration_secret: "{{ pg_config['resources'][0]['metadata']['name'] }}"
+
+- name: Store Database Configuration
     pg_config: '{{ _generated_pg_config_resources["resources"] | default([]) | length | ternary(_generated_pg_config_resources, _pg_config) }}'
   no_log: "{{ no_log }}"
 
 - name: Set actual postgres configuration secret used
-  set_fact:
-    __postgres_configuration_secret: "{{ pg_config['resources'][0]['metadata']['name'] }}"
-
-- name: Store Database Configuration
   set_fact:
     awx_postgres_user: "{{ pg_config['resources'][0]['data']['username'] | b64decode }}"
     awx_postgres_pass: "{{ pg_config['resources'][0]['data']['password'] | b64decode }}"
     awx_postgres_database: "{{ pg_config['resources'][0]['data']['database'] | b64decode }}"
     awx_postgres_port: "{{ pg_config['resources'][0]['data']['port'] | b64decode }}"
     awx_postgres_host: "{{ pg_config['resources'][0]['data']['host'] | b64decode }}"
     awx_postgres_sslmode: "{{ pg_config['resources'][0]['data']['sslmode'] |  default('prefer'|b64encode) | b64decode }}"
+    awx_postgres_sslunauthorized: "{{ pg_config['resources'][0]['data']['sslrejectUnauthorized'] |  default('false'|b64encode) | b64decode }}"
```

Comment:
do you mind adding a node about this setting to the operator docs, somewhere here

https://github.com/ansible/awx-operator/blob/6513e93e7fdf117a1518392a0d838f267a67088d/README.md?plain=1#L536

you don't need to change the example spec above that line though, just a note about this param

Reply:
Done @fosterseth also default value i have set it to false do you think it has to be set to true and then if some one uses external database without sslrejectUnauthorized they can pass the variable false.